ui/app/rif/components/lumino/open-channel/index.js

```diff
@@ -206,7 +208,7 @@ class OpenChannel extends Component {
       if (this.state.amount) {
         this.setState({
           loading: true,
-          loadingMessage: 'Making deposit...',
+          loadingMessage: 'Making deposit\nThis operation can take around 6 minutes',
```

Comment:
I think a better way to say that could be 'Making deposit.\n Please wait, this operation could take about 6 minutes.', but it's just a suggestion.